.gitignore

```diff
@@ -6,3 +6,4 @@ build
 .project
 .settings
 *.iml
+.vscode
```

Comment:
You're right here in that we probably don't want to add he .vscode dir into out repo. Maybe i should just document what should be included in the readme for PR #43, and also add the .vscode to the gitignore there.

Reply:
I added it before looking at your PR. Since it contains only 1 json file I'm not bothered by it.

Reply:
Do we want to remove it from the gitignore, or the repo then?